internal/plugins/ansible/v1/scaffolds/internal/templates/requirements.go

```diff
@@ -34,7 +34,6 @@ func (f *RequirementsYml) SetTemplateDefaults() error {
 const requirementsYmlTmpl = `---
 collections:
   - name: community.kubernetes
-    version: "1.1.1"
-  - name: operator_sdk.util
-    version: "0.1.0"
```

Comment:
Why did you remove this `version`?

Reply:
I just found this too, probably it was just a non-properly resolved merge conflict as that changed was recently merged in #4529.